force_wfmanager/wfmanager_setup_task.py

```diff
@@ -135,83 +146,19 @@ class WfManagerSetupTask(Task):
-                    enabled_name="save_load_enabled",
-                    accelerator="Ctrl+O",
-                ),
-                TaskAction(
-                    id="Save",
-                    name="Save Workflow",
-                    method="save_workflow",
-                    enabled_name="save_load_enabled",
-                    accelerator="Ctrl+S",
-                ),
-                TaskAction(
-                    name="Save Workflow as...",
-                    method="save_workflow_as",
-                    enabled_name="save_load_enabled",
-                    accelerator="Shift+Ctrl+S",
-                ),
-                TaskAction(name="Plugins...", method="open_plugins"),
-                TaskAction(name="Exit", method="exit"),
-                # NOTE: Setting id='File' here will automatically create
-                #       a exit menu item, I guess this is QT being 'helpful'.
-                #       This menu item calls application.exit, which bypasses
-                #       our custom exit which prompts for a save before exiting
-                name="&File",
-            ),
-            SMenu(
-                TaskAction(
-                    name="About WorkflowManager...", method="open_about"
-                ),
-                name="&Help",
-            ),
-            SMenu(TaskToggleGroupAccelerator(), id="View", name="&View"),
-        )
-        return menu_bar
+        return SMenuBar(id='mymenu')
 
     def _tool_bars_default(self):
         return [
             SToolBar(
-                TaskAction(
-                    name="Run",
-                    tooltip="Run Workflow",
-                    image=ImageResource("baseline_play_arrow_black_48dp"),
-                    method="run_bdss",
-                    enabled_name="run_enabled",
-                    image_size=(64, 64),
-                ),
-                TaskAction(
-                    name="Stop",
-                    tooltip="Stop Workflow",
-                    method="stop_bdss",
-                    enabled_name="computation_running",
-                    image=ImageResource("baseline_stop_black_18dp"),
-                    image_size=(64, 64),
-                ),
-                TaskAction(
-                    name=" Pause",
-                    tooltip="Pause Workflow",
-                    method="pause_bdss",
-                    enabled_name="computation_running",
-                    image=ImageResource("baseline_pause_black_18dp"),
-                    image_size=(64, 64),
-                ),
                 TaskAction(
                     name="View Results",
                     tooltip="View Results",
                     image=ImageResource("baseline_bar_chart_black_48dp"),
                     method="switch_task",
                     image_size=(64, 64),
-                ),
+                    id='View Results'
+                ), id='running'
             ),
             SToolBar(
                 TaskAction(
```

Comment:
Some of these `TaskAction` items can be removed as they are now already defined in `WfManagerGlobalTask`

Reply:
@sparsonslab - did you check this out? I think the comment got lost amongst the sea of application exit talk